composer.json

```diff
@@ -26,7 +32,7 @@
         "ext-SimpleXML": "*",
         "ext-openssl": "*",
         "ext-gettext": "*",
-        "mapkyca/php-ogp": "^1.0",
+        "mapkyca/php-ogp": "dev-main#b24efe2738d367472f2810446fad6e4b238875a8",
```

Comment:
I would prefer to use the official version when your patch lands in the php-ogp repo.

Reply:
Indeed, I'd rather use a tag as well. 
This commit hash was added to let you known beforehand with the `mapkyca/php-ogp` library version matching this PR.
I didn't make it clear enough that it would require a rework i.e. setting a proper version for this dependency as soon as the related changes would be merged at https://github.com/mapkyca/php-ogp/pull/4

Reply:
@mapkyca can you review the php-ogp PR and bump the version?

Reply:
Merged. Dependabot should pick up the changes shortly...

Reply:
Great!